third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -1130,15 +1834,43 @@ void initNvFuserPythonBindings(PyObject* module) {
 #define NVFUSER_PYTHON_BINDING_REDUCTION_OP(op_str, op_name)                            \
   nvf_ops.def(                                                                          \
       op_str,                                                                           \
-      [](FusionDefinition::Operators& self,                                             \
-         Tensor arg,                                                                    \
+      [](FusionDefinition::Operators& self,                                             \
+         Tensor arg,                                                                    \
+         const std::vector<int>& axes,                                                  \
+         bool keepdim,                                                                  \
+         PrimDataType dtype) -> Tensor {                                                \
+        FUSER_PERF_SCOPE("Operators." op_str);                                          \
+        TORCH_CHECK(                                                                    \
+            self.validUse(), "Attempting to add to a completed definition!");           \
+        FusionDefinition* fd = self.fusion_definition;                                  \
+        size_t ndims = keepdim ? arg.dims : (arg.dims - axes.size());                   \
+        Tensor output = fd->defineTensor(ndims);                                        \
+        fd->defineRecord(new ReductionOpRecord(                                         \
+            {fd->recordingState(arg())},                                                \
+            {fd->recordingState(output())},                                             \
+            ("ops." op_str),                                                            \
+            static_cast<                                                                \
+                                                                                        \
+                TensorView* (*)(TensorView*, const std::vector<int>&, bool, DataType)>( \
```

Comment:
nitpick, this is some strange formatting. clangformat is beyond my comprehension.

Reply:
I noticed it and have no clue why clang-format insists on that formatting.